src/Application/Models/Bank/Asset.js

```diff
@@ -39,6 +40,14 @@ class Asset {
     this._multiplier += additionalPercentage;
   }
 
+  unlock() {
+    this._isUnlocked = true;
+  }
```

Comment:
I'm going to be moving all of the Model directory stuff into an immutable state machine kind of thing with Redux, so this will be taken care of in the commit after this one.